mam/mam/endpoint.c

```diff
@@ -73,33 +93,46 @@ retcode_t mam_endpoints_destroy(mam_endpoint_t_set_t *const endpoints) {
 }
 
 size_t mam_endpoint_serialized_size(mam_endpoint_t const *const endpoint) {
-  return pb3_sizeof_size_t(trits_size(endpoint->name)) +                                            // name size
+  return pb3_sizeof_ntrytes(MAM_TRITS_MAX_SIZEOF_SIZE_T / NUMBER_OF_TRITS_IN_A_TRYTE) +             // name size
```

Comment:
same, why not `mam_endpoint_name_size` instead of `MAM_TRITS_MAX_SIZEOF_SIZE_T`?